src/Orleans.Runtime/GrainDirectory/ClientObserverRegistrar.cs

```diff
@@ -54,7 +54,7 @@ internal void SetHostedClient(IHostedClient client)
             this.hostedClient = client;
             if (client != null)
             {
-                this.scheduler.QueueAction(Start, this).Ignore();
+                this.scheduler.QueueAction(Start, this);
```

Comment:
Note that this method is now implemented on OrleansTaskScheduler itself rather than as an extension. It is a void method now.
The extension method which this line previously called (which returns a Task) was renamed to `QueueActionAsync`.